lib/wsm-panel.js

```diff
@@ -228,6 +228,8 @@ const WsmPanel = Class(
   // Socket.IO Parser
 
   decodeSocketIoPacket: function(data) {
+    if (!prefs.enableSocketIo) return;
```

Comment:
Yes

One of the long term goals is to integrate ws-monitor directly into DevTools and so, we might want to use the existing `.eslintrc` so, the code is compatible and future merge is easier.

Honza